handler.go

```diff
@@ -62,5 +63,6 @@ func newTransportConfig(hConfig HandlerConfig) transportConfig {
 		webhookLegacyDecodeCount: hConfig.MetricsProvider.NewCounter(WebhookLegacyDecodeCount),
 		now:                      time.Now,
 		v:                        hConfig.V,
+		reqPartnerIDs:            hConfig.reqPartnerIDs,
```

Comment:
As a general rule, try to avoid abbreviations in configuration values that are set externally, such as in a `yaml` file.  A person unfamiliar with the codebase has more mental friction looking at `reqPartnerIDs` as opposed to `requirePartnerIDs`.  It's also consistent with our other configuration names.